typeset/mjnode.js

```diff
@@ -1,23 +1,14 @@
-const mjAPI = require('mathjax-node')
+let mjAPI;
 
-const convertMathML = async (log, mathMap/*: Map<string, {xml: string, fontSize: number} */, outputFormat) => {
+const startAPI = async (log) => {
```

Comment:
What is the benefit to this function being async anymore? It doesn't look like any async code or callback is being run in it. Still kind of a noob to async functions. @philschatz might also have an answer for me.

Reply:
@tomjw64 I'm very new to this :sweat_smile: so now that you point it out, I don't think we need `async` here.  I'll try removing it.